code/modules/client/preferences.dm

```diff
@@ -1969,7 +1969,7 @@ GLOBAL_LIST_EMPTY(preferences_datums)
 
 	var/datum/species/chosen_species
 	chosen_species = pref_species.type
-	if(!roundstart_checks || (pref_species.id in GLOB.roundstart_races))
+	if(!roundstart_checks || (pref_species.id in GLOB.roundstart_races) || (pref_species.id in GLOB.roundstart_no_hard_check))
```

Comment:
I don't see why we need a global list for something that will rarely be checked, you can probably just get away with calling the proc here

Reply:
Seems to work with just calling the proc and without the list. I changed it accordingly.